addon/components/new-osf-navbar/style.scss

```diff
@@ -0,0 +1,228 @@
+// Overrides some of base osf-styles for the navbar in osf-style/sass/_bootstrap.scss
+$osf-teal-navbar: #2E6A74;
+$osf-highlight-navbar: #5A8288;
+$osf-dark-blue-navbar: #263947;
+
+.new-osf-navbar-styling {
+    /* Overrides the overall navbar color */
+    .navbar-inverse {
+        background-color: $osf-dark-blue-navbar;
+    }
+
+    /* Enlarges OSF Font */
+    .navbar-brand {
+        font-size: 28px;
+    }
+
+    /* Enlarges OSF logo */
+    .osf-navbar-logo {
+        width: 35px;
+        height: 35px;
+        margin-top: -4px;
+        margin-left: -4px;
+    }
+
+    /* Defines position of primary navigation: dropdown toggle (caret) */
+    .primary-nav {
+        float: left;
+        /* Overrides vertical and horizontal position of dropdown caret for service menu */
+        li.dropdown {
+            margin-left: -46px;
+            padding-top: 15px;
+            list-style-type: none;
+            font-size: 15px;
+        }
+    }
+
+    /* Overrides the color, relative position, and font size of dropdown menus (both service and auth) */
+    .dropdown-menu {
+        background-color: $osf-teal-navbar;
+        &.service-dropdown {
+            li > a {
+               font-size: 16px;
+            }
+            top: 58px;
+            left: -14.5px;
+        }
+        &.auth-dropdown {
+            li > a {
+               font-size: 15px;
+            }
+            top: 60px;
+            left: 15px;
+        }
+    }
+
+    /* Overrides background of dropdown-toggle when active - needed for auth navbar */
+     .navbar-nav > .open > a,
```

Comment:
Thanks. It appears that at present the consuming app has to install `osf-style` itself; when that time comes, this may take some coordination across consuming repos to update `osf-style`.
https://github.com/abought/demo-ember-osf/blob/develop/ember-cli-build.js#L49
https://github.com/abought/demo-ember-osf/blob/develop/bower.json

(it's definitely one of the pain points of using the addon at present- styles are pinned to a specific version, and the dependency version is defined independently in many different apps)